packages/api/test/e2e/promise-tx.spec.ts

```diff
@@ -138,38 +144,37 @@ describe.skip('e2e transactions', () => {
   it('makes a transfer, and uses new balance to transfers to new', async (done) => {
     const pair = new Keyring().addFromUri('testing123', {}, 'ed25519');
 
-    function doOne (cb) {
+    function doOne (cb: any) {
       return api.tx.balances
         .transfer(pair.address(), 123456)
         .signAndSend(keyring.dave, logEvents(cb));
-    };
+    }
 
-    function doTwo (cb) {
+    function doTwo (cb: any) {
       return api.tx.balances
         .transfer(keyring.alice.address(), 12345)
         .signAndSend(pair, logEvents(cb));
     }
 
     // return doTwo(done);
     return doOne(() => {
-      doTwo(done)
+      doTwo(done).catch();
     });
   });
 
   it('makes a transfer with ERA (signAndSend)', async (done) => {
     const nonce = await api.query.system.accountNonce(keyring.dave.address());
     const signedBlock = await api.rpc.chain.getBlock();
-    const currentHeight = signedBlock.block.header.number;
+    const currentHeight = (signedBlock as SignedBlock).block.header.number;
     const exERA = new ExtrinsicEra({ current: currentHeight, period: 10 });
     // eraBirth - start of ERA which is always less than current block height
     // eraDeath - end of ERA validity (EXPIRY)
     const eraBirth = exERA.asMortalEra.birth(currentHeight.toNumber());
     const eraDeath = exERA.asMortalEra.death(currentHeight.toNumber());
-    console.log('STARTED AT :'+eraBirth+' EXPIRED AT :'+eraDeath);
+    console.log('STARTED AT :' + eraBirth + ' EXPIRED AT :' + eraDeath);
     const eraHash = await api.rpc.chain.getBlockHash(eraBirth);
-    const ex = api.tx.balances
-      .transfer(keyring.eve.address(), 12345);
-    const hash = await ex.signAndSend(keyring.dave, {blockHash: eraHash, era:exERA, nonce});
+    const ex = api.tx.balances.transfer(keyring.eve.address(), 12345);
+    const hash = await ex.signAndSend(keyring.dave, { blockHash: eraHash, era: exERA, nonce });
```

Comment:
@amaurymartiny any Idea here? I'm getting:

```
Argument of type '{ blockHash: Codec; era: ExtrinsicEra; nonce: Codec; }' is not assignable to parameter of type 'StatusCb'.
  Object literal may only specify known properties, and 'blockHash' does not exist in type 'StatusCb'.
```

Reply:
Not sure here, you can just `{ blockHash: eraHash, era: exERA, nonce } as any`

Reply:
ok, that is working. thanks!